pom.xml

```diff
@@ -128,7 +128,7 @@
     <slf4j.version>1.7.10</slf4j.version>
     <log4j.version>1.2.17</log4j.version>
     <hadoop.version>2.2.0</hadoop.version>
-    <protobuf.version>2.5.0</protobuf.version>
+    <protobuf.version>2.6.1</protobuf.version>
```

Comment:
Agreed... definitely needs testing.

Just upgrading the KCL didn't work.  The new KCL relies on the following class from protobuf:  `com/google/protobuf/ProtocolStringList.class`, which doesn't exist until 2.6.1.